CauldronMods/Controller/Heroes/TheStranger/Cards/TheOldRoadsCardController.cs

```diff
@@ -6,27 +6,22 @@
 
 namespace Cauldron.TheStranger
 {
-    public class TheOldRoadsCardController : CardController
+    public class TheOldRoadsCardController : TheStrangerBaseCardController
     {
-        #region Constructors
-
         public TheOldRoadsCardController(Card card, TurnTakerController turnTakerController) : base(card, turnTakerController)
         {
-            SpecialStringMaker.ShowNumberOfCardsAtLocation(TurnTaker.Trash, new LinqCardCriteria(c => IsGlyph(c), "glyph"));
-            SpecialStringMaker.ShowNumberOfCardsAtLocation(TurnTaker.Deck, new LinqCardCriteria(c => IsGlyph(c), "glyph"));
+            SpecialStringMaker.ShowNumberOfCardsAtLocation(TurnTaker.Trash, IsGlyphCriteria());
+            SpecialStringMaker.ShowNumberOfCardsAtLocation(TurnTaker.Deck, IsRuneCriteria());
```

Comment:
```suggestion
            SpecialStringMaker.ShowNumberOfCardsAtLocation(TurnTaker.Deck, IsGlyphCriteria());
```

Reply:
Both strings should be Glyph.  Might be slightly better to just ShowCardsAtLocations override, but I missed this guy either way.
![image](https://user-images.githubusercontent.com/7707342/104828016-e47df080-5832-11eb-8649-6ca046075dbd.png)